cmd/operator-sdk/test/local.go

```diff
@@ -74,7 +76,13 @@ func newTestLocalCmd() *cobra.Command {
 	testCmd.Flags().StringVar(&tlConfig.moleculeTestFlags, "molecule-test-flags", "",
 		"Additional flags to pass to molecule test")
 	testCmd.Flags().StringVar(&tlConfig.namespace, "namespace", "",
-		"If non-empty, single namespace to run tests in")
+		"(Deprecated: use --operator-namespace instead) If non-empty, single namespace to run tests in")
+	testCmd.Flags().StringVar(&tlConfig.operatorNamespace, "operator-namespace", "",
+		"Namespace where the operator will be deployed, CRs will be created and tests will be executed "+
```

Comment:
```suggestion
		"Namespace where the operator will be deployed"+
```


The CR's are not created by the operator in the default implementation. It needs to be applied manually.

Reply:
@camilamacedo86 
in operator-sdk test local --up-local the CRs are programmatically  created by users who write tests right.
whereas  in operator-sdk run --local a user will have to add the CRDs, CRS, RBAC etc manually, right